src/odemis/dataio/tiff.py

```diff
@@ -3131,6 +3133,7 @@ def _convert_tescan_to_odemis_metadata(metadata: bytes) -> dict:
             md[model.MD_POS] = (float(tmd[beam_type]["stagex"]),
                                 float(tmd[beam_type]["stagey"]),
                             )
+            md[model.MD_ROTATION] = 0 # float(tmd[beam_type]["stager"])
```

Comment:
Let's drop this. No need to (not?) associate it with "stage/r".
If you want, you could extend the metadata of about the stage position to be stored in MD_STAGE_POSITION_RAW (as a dict with "rz").